Makefile.am

```diff
@@ -153,6 +153,7 @@ checkbashisms:
 				-o -name 'config' -prune \
 				-o -name 'zed-functions.sh*' -prune \
 				-o -name 'zfs-import*' -prune \
+				-o -name 'zfs-load-key*' -prune \
```

Comment:
If you drop the `local` decls (which aren't really necessary, considering this isn't a library), you can drop this exclusion

Reply:
I have oriented me on the other scripts. Should the other scripts be updated to? Maybe this can be done as separte task?

Reply:
You've oriented yourself on ancient versions of those scripts. Rebase on `master` and orient yourself accordingly.